src/consensus/params.rs

```diff
@@ -34,6 +34,13 @@ const MAX_BITS_TESTNET: Uint256 = Uint256([
     0xffffffffffffffffu64,
     0x00000000ffffffffu64,
 ]);
+/// Lowest possible difficulty for Signet.
+const MAX_BITS_SIGNET: Uint256 = Uint256([
+    0x62a429192f9e8ce5u64,
+    0xc63b5f100aa5d8dfu64,
+    0x3c82faa55d83e40au64,
+    0x00002adc28cf53b6u64,
```

Comment:
These parameters can be verified here: https://github.com/kallewoof/bitcoin/blob/2b9db9ac0936cf278ad27c59497d9feec61f6440/src/chainparams.cpp#L306